src/main/com/fulcrologic/rad/routing/html5_history.cljc

```diff
@@ -91,7 +91,9 @@
         (history/-push-route! this route params))))
   (-back! [_]
     #?(:cljs
-       (.back js/history)))
+       (let []
+         (log/spy :debug ["Back to prior route" (some-> prior-route deref)])
```

Comment:
leaving this here for further debugging?

Reply:
Yeah, the other history API calls had debug logs, but not this one. Made it hard to figure out this was the culprit.

Reply:
The other debug logs were just like this spy.